build/ssr.js

```diff
@@ -1,34 +1,34 @@
-var rollup = require('rollup')
-var buble = require('rollup-plugin-buble')
-var async = require('rollup-plugin-async')
-var replace = require('rollup-plugin-replace')
+var rollup = require('rollup');
+// var buble = require('rollup-plugin-buble');
+// var async = require('rollup-plugin-async')
+var replace = require('rollup-plugin-replace');
 
 rollup
   .rollup({
     input: 'packages/docsify-server-renderer/index.js',
     plugins: [
-      async(),
+      //   async(),
       replace({
         __VERSION__: process.env.VERSION || require('../package.json').version,
-        'process.env.SSR': true
+        'process.env.SSR': true,
       }),
-      buble({
-        transforms: {
-          generator: false
-        }
-      })
+      //   buble({
+      //     transforms: {
+      //       generator: false,
+      //     },
+      //   }),
```

Comment:
The output is nice without this. async functions and generators work great in newer Node.